config/webpack.common.js

```diff
@@ -17,8 +17,12 @@ const betaBranch =
     gitBranch === 'master' ||
     gitBranch === 'qa-beta' ||
     gitBranch === 'prod-beta';
+const buildStable = process.env.BUILD_STABLE === 'true';
 
-if ((process.env.NODE_ENV === 'production' && betaBranch) || process.env.BUILD_BETA === 'true') {
+const buildBeta = (betaBranch || process.env.BUILD_BETA === 'true') && !buildStable;
+
+if (process.env.NODE_ENV === 'production' && buildBeta) {
+    console.log('building beta');
```

Comment:
just confirming, this log statement is intentional and not debug, correct?

Reply:
Good catch, dropped.